modules/platform/nuxeo-platform-document-routing/nuxeo-routing-core/src/main/java/org/nuxeo/ecm/platform/routing/core/api/operation/StartWorkflowOperation.java

```diff
@@ -97,6 +98,10 @@ public DocumentModel run(DocumentModel doc) {
     }
 
     protected void startNewInstance(List<String> ids) {
+        if (!documentRoutingService.canCreateInstance(session, ids, id)) {
+            throw new NuxeoException(
+                    String.format("Cannot start the the worflow model %s for the input documents", id));
```

Comment:
`the the` ->`the`
Maybe log the `ids` too in the exception?

Reply:
done